web/impact/impact/tests/test_impact_email_backend.py

```diff
@@ -0,0 +1,63 @@
+from mock import patch
+from django.core import mail
+from django.test import TestCase
+from django.urls import reverse
+
+from impact.minimal_email_handler import MinimalEmailHandler
+
+
+IMPACT_BACKEND_PATH = 'impact.impact_email_backend.ImpactEmailBackend'
```

Comment:
Why are we using this value instead of getting it from `settings.EMAIL_BACKEND`?

Reply:
Good thought

Reply:
@pbx I attempted to make this change, but it caused the tests to fail. How critical is this for you? I will spend more time on it if you think it's important that I do so.

Reply:
Sorry it's been so time-consuming, but yeah, I do think it's important.  I like the idea of a minimal email handler that will serve as the basis for future expansion, but I want to make sure it's right and that it's defended against accidental breakage later. Thanks.